docs/ops/deployment/kubernetes.md

```diff
@@ -76,57 +74,92 @@ You can then access the Flink UI via different ways:
     {% highlight bash %}./bin/flink run -m localhost:8081 ./examples/streaming/WordCount.jar{% endhighlight %}
 
 *  Create a `NodePort` service on the rest service of jobmanager:
-    1. Run `kubectl create -f jobmanager-rest-service.yaml` to create the `NodePort` service on jobmanager. The example of `jobmanager-rest-service.yaml` can be found in [appendix](#session-cluster-resource-definitions).
+    1. Run `kubectl create -f jobmanager-rest-service.yaml` to create the `NodePort` service on jobmanager. The example of `jobmanager-rest-service.yaml` can be found in [appendix](#common-cluster-resource-definitions).
     2. Run `kubectl get svc flink-jobmanager-rest` to know the `node-port` of this service and navigate to [http://&lt;public-node-ip&gt;:&lt;node-port&gt;](http://<public-node-ip>:<node-port>) in your browser.
-    3. Similarly to `port-forward` solution, you could also use the following command below to submit jobs to the cluster:
+    3. If you use minikube, you can get its public ip by running `minikube ip`.
+    4. Similarly to `port-forward` solution, you could also use the following command below to submit jobs to the cluster:
 
         {% highlight bash %}./bin/flink run -m <public-node-ip>:<node-port> ./examples/streaming/WordCount.jar{% endhighlight %}
 
-In order to terminate the Flink session cluster, use `kubectl`:
+In order to terminate the Flink cluster, terminate the specific [Session](#deploy-session-cluster) or [Job](#deploy-job-cluster) cluster components
+and use `kubectl` to terminate the common components:
 
-    kubectl delete -f jobmanager-deployment.yaml
-    kubectl delete -f taskmanager-deployment.yaml
+```sh
     kubectl delete -f jobmanager-service.yaml
     kubectl delete -f flink-configuration-configmap.yaml
+    # if created then also the rest service
+    kubectl delete -f jobmanager-rest-service.yaml
+```
+
+### Deploy Session Cluster
+
+A Flink Session cluster is executed as a long-running Kubernetes Deployment.
+Note that you can run multiple Flink jobs on a Session cluster.
+Each job needs to be submitted to the cluster after the cluster has been deployed.
 
-## Flink job cluster on Kubernetes
+A basic Flink Session cluster deployment in Kubernetes has three components:
```

Comment:
What does an advanced Flink Session cluster look like?

Reply:
I think it implicitly means here that this is just a simple example to start from.